packages/@uppy/companion/src/standalone/index.js

```diff
@@ -11,25 +11,20 @@ const connectRedis = require('connect-redis')
 const logger = require('../server/logger')
 const redis = require('../server/redis')
 const companion = require('../companion')
-const helper = require('./helper')
+const { getCompanionOptions, generateSecret, buildHelpfulStartupMessage } = require('./helper')
 
 /**
  * Configures an Express app for running Companion standalone
  *
  * @returns {object}
  */
-module.exports = function server (inputCompanionOptions = {}) {
-  let corsOrigins
-  if (process.env.COMPANION_CLIENT_ORIGINS) {
-    corsOrigins = process.env.COMPANION_CLIENT_ORIGINS
-      .split(',')
-      .map((url) => (helper.hasProtocol(url) ? url : `${process.env.COMPANION_PROTOCOL || 'http'}://${url}`))
-  } else if (process.env.COMPANION_CLIENT_ORIGINS_REGEX) {
-    corsOrigins = new RegExp(process.env.COMPANION_CLIENT_ORIGINS_REGEX)
-  }
+module.exports = function server (inputCompanionOptions) {
+  const companionOptions = getCompanionOptions(inputCompanionOptions)
+
+  companion.setLoggerProcessName(companionOptions)
 
-  const moreCompanionOptions = { ...inputCompanionOptions, corsOrigins }
-  const companionOptions = helper.getCompanionOptions(moreCompanionOptions)
+  if (!companionOptions.secret) companionOptions.secret = generateSecret()
```

Comment:
```suggestion
  companionOptions.secret ??= generateSecret()
```
or (if we don't want to accept empty string as secret):
```suggestion
  companionOptions.secret ||= generateSecret()
```